dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/Connector.java

```diff
@@ -36,10 +38,19 @@ public enum DefaultProperties implements ConnectorProperty {
     public String getName();
 
     @Nonnull
-    public String getDefaultFileName();
+    default String getDefaultFileName() {
+        SimpleDateFormat format = new SimpleDateFormat("yyyyMMdd'T'HHmmss");
```

Comment:
nit: I would recommend to change the type of the variable to `DateFormat`.

Reason: Item 64: Refer to objects by their interfaces, Effective Java (the `DateFormat` is an abstract class, not an interface, but I would use the same rationale here)

Reply:
Used the `Format` interface directly 😄

Reply:
The alternative could be to use `ISO_LOCAL_DATE_TIME`.
https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME

See example usage in existing code:
https://github.com/google/dwh-migration-tools/search?q=DateTimeFormatter

Reply:
I wanted to get rid of all the `-`-s. Do let me know if you would prefer the  `ISO_LOCAL_DATE_TIME`

Reply:
Makes sense. I didn't notice the `-` and `:` separators. Thank you for checking.